playbook-website/app/javascript/components/Website/src/pages/ComponentShow.tsx

```diff
@@ -40,6 +40,7 @@ export default function ComponentShow() {
           },
         }}
         options={{
+          bundlerURL: "https://pr3210.playbook.beta.gm.powerapp.cloud/www/index.html",
```

Comment:
This will change. Just wanting to see if testing in a milano environment was even possible.